atomics/T1059.002/T1059.002.md

```diff
@@ -1,46 +0,0 @@
-# T1059.002 - Command and Scripting Interpreter: AppleScript
-## [Description from ATT&CK](https://attack.mitre.org/techniques/T1059/002)
-<blockquote>Adversaries may abuse AppleScript for execution. AppleScript is a macOS scripting language designed to control applications and parts of the OS via inter-application messages called AppleEvents.(Citation: Apple AppleScript) These AppleEvent messages can be sent independently or easily scripted with AppleScript. These events can locate open windows, send keystrokes, and interact with almost any open application locally or remotely.
```

Comment:
Thanks for your first contribution. Welcome to the project 😄 Can you undelete this file ?

Reply:
My apologies, my antivirus really doesn't like that file (there seems to be some bash scripts inside of the markdown). Should be good to go! Also, thanks for the warm welcome, glad to be contributing!

Reply:
Haha no worries. Thanks for the quick fix.